server/src/main/java/org/elasticsearch/index/mapper/TsidExtractingIdFieldMapper.java

```diff
@@ -156,13 +156,25 @@ public void createField(DocumentParserContext context, BytesRef tsid) {
     }
 
     @Override
-    public String documentDescription(SourceToParse source) {
+    public String documentDescription(DocumentParserContext context) {
         /*
          * We don't yet have an _id because it'd be generated by the document
-         * parsing process. And the _id isn't all that useful anyway - what
-         * folks really want to to see is the dimensions and timestamp. And
-         * those too aren't available until after we've parsed the document.
+         * parsing process. But we *might* have something more useful - the
+         * time series dimensions and the timestamp! If we have those, then
+         * include them in the description. If not, all we know is
+         * "a time series document".
          */
-        return "a time series document";
+        StringBuilder description = new StringBuilder("a time series document");
+        IndexableField tsidField = context.doc().getField(TimeSeriesIdFieldMapper.NAME);
+        if (tsidField != null) {
+            String tsid = TimeSeriesIdFieldMapper.decodeTsid(tsidField.binaryValue()).toString();
+            description.append(" with dimensions ").append(tsid);
```

Comment:
@henningandersen and I talked about limiting this to maybe 1k characters or something. I hate that it'd make debugging hard but it's nice defense against a *huge* bulk response.